src/services/createRouter.ts

```diff
@@ -309,6 +311,10 @@ export function createRouter<
   }
 
   function install(app: App): void {
+    vueApp = app
+
+    propStore.setVueApp(app)
```

Comment:
I know this is overkill but thoughts on storing the vueApp in it's own "store"? That would solve 2 things I don't love about the current implementation. 

1. we use a `let`
2. the hooks and props technically have independent references and ideally they'd look in the same place

Reply:
that new "store" could also be responsible for exporting the `runWithContext` piece

Reply:
Don't hate it, I'll look into that tomorrow.

Reply:
@stackoverfloweth looking at this again not sure how the new store would work. We don't really have a way for the propsStore to get something from a separate store. Maybe we can chat about this

Reply:
We might abstract this further in the future. But for now we're going to keep this simple.